README.md

```diff
@@ -30,18 +30,39 @@ $ ember exam --filter='acceptance'
 $ ember exam --server
 ```
 
+A value to an option can be passed with either `=` and a space.
+```bash
+# A value of filter is acceptance
+$ ember exam --filter 'acceptance'
+
+# A value of parallel is 2
+$ ember exam --load-balance --parallel=2 --server --no-launch
+
+# A value of parallel is --server
+$ ember exam --load-balance --parallel --server --no-launch
+```
+
 The idea is that you should be able to replace `ember test` with `ember exam` and never look back.
 
-However, to get the unique features of Ember Exam (described in-depth below), you will need to load Ember Exam before your test suite runs. This is done by invoking the `loadEmberExam` function, usually from within `test-helper.js` but it can be done anywhere that executes prior to your tests loading.
+However, to get the unique features of Ember Exam (described in-depth below), you will need to import addon-test-support/load.js to invoke the loadEmberExam function. This is done usually from within test-helper.js but it can be done anywhere that executes prior to your tests loading.
 
 ```js
 // test-helper.js
-import loadEmberExam from 'ember-exam/test-support/load';
+import emberExam from 'ember-exam/test-support/load';
+
+emberExam.loadEmberExam(); // To set an instance of testLoader and pass it to testem
+
+start();
```

Comment:
where does start() come from

Reply:
the start() used to come from ember-qunit to kick off QUnit. After the simplification of ember exam setup (ref. https://github.com/ember-cli/ember-exam/pull/136/commits/207fa49db1c429189887c19c9209c23ece2275fe#diff-412f33e8784be93672251a0e3b3f9ab9R1) it's now only required to call ember-exam's own start function.